_internals/tds-protocol.md

```diff
@@ -85,8 +85,27 @@ Once we are over the <code>PRELOGIN</code> phase, we are ready for
 <code>LOGIN7</code> which defines the authentication rules for use 
 between client and server.
 
-
-### Defining authentication rules.
+### SSL/TLS support for TDS connections
+
+Babelfish uses native PostgreSQL support for using SSL connections to encrypt client/server communications. See PostgresSQL [documentation](https://www.postgresql.org/docs/13/ssl-tcp.html#SSL-SETUP) for details about configuring server-side SSL functionality.
```

Comment:
I see that this has been pushed, but there is a typo in the above line... PostgreSQL is misspelled.

Reply:
No Susan, the commit wasn't pushed in the main branch. I've only pushed it in a separate branch. Once I fix all the review comments and get approvals, I'll merge the changes.

Reply:
Sorry, my bad!  Let me try again:

Babelfish uses native PostgreSQL functionality to support SSL connections that encrypt client/server communications. See the PostgreSQL [documentation](https://www.postgresql.org/docs/13/ssl-tcp.html#SSL-SETUP) for details about configuring server-side SSL functionality.

Reply:
[Fixed](https://github.com/babelfish-for-postgresql/babelfish_project_website/pull/145/commits/d6ef975eeca30403245ee37cb8299e18e54c51e4)